app/modules/asset_groups/models.py

```diff
@@ -883,7 +883,9 @@ class AssetGroup(db.Model, HoustonModel):
         cascade='all, delete',
     )
 
-    assets = db.relationship('Asset', back_populates='asset_group', order_by='Asset.guid')
+    # assets = db.relationship(
```

Comment:
Curious why this is commented out, that should be used IIRC.

Reply:
It was commented out because the `back_populates` was converted into a `backref` on the AssetGroup model.  The need for this change is so that asset groups can be disabled (not needed for MWS) and the relationship is only defined with annotations when it is enabled.  We need to come up with a standardized way to document backrefs.  The previous attempt to convert these relationships to backrefs left commented out attributes in their old place so we wouldn't forget about it.

This PR may be the right time to figure out how to document these backrefs and the attributes they imply when enabled.